defect-tracker-server/src/main/java/com/defect/tracker/services/LoginServiceImpl.java

```diff
@@ -131,4 +131,20 @@ public String emailVerification(String token, String email) {
 		return response ;
 	}
 
+
+	@Override
+	public void create(Login login) {
+		loginRepository.save(login);
+		
+	}
+
+	@Override
+	public void updateEmployeeStatus(Login login) {
+		// TODO Auto-generated method stub
```

Comment:
if you do not need this method remove it

Reply:
removed in the next commit